types/src/transaction/metadata.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
We don't need this file any more, do we?

If you want, we can also generate such a file next to `libra.yaml`. The problem here is that consistency between `metadata.rs` and `metadata.yaml` is not guaranteed by CI.

Reply:
Woops yep this is actually a legacy, removing that